README.md

```diff
@@ -23,9 +23,9 @@ Accepted Arguments
  * AK - Name of the Activation Key to use.
  * AUTH - Satellite username and password. (AUTH=username/password)
  * ENV - Name of the Environment to use.
- * KILL - If this is not passed, then the container will be kept alive and goferd running.
- * ORG - Name of the Organization to use.
- * SATHOST(Required) - Hostname of the Satellite (not url).
+ * KILL - Life in seconds. Omit to leave container and goferd running.
+ * ORG - The label of the Organization to use.
+ * SATHOST(Required) - Host name of the Satellite (not url).
```

Comment:
KILL and SATHOST were accurate

Reply:
The name of a host on the network is the host name. HOSTNAME is a directive in a config file, **hostname** is a command to list the host name. I have made some changes, please review once more. Thank you.